libs/io/tests/test_h5.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# coding: utf-8
+import os
+import shutil
+from pathlib import Path
+
+from typing import TYPE_CHECKING, Optional, Tuple
+import h5py
+import pytest
+import numpy as np
+
+
+from bbhnet.io import h5
+
+#from h5 import write_timeseries
+#from h5 import read_timeseries
+
+@pytest.fixture(scope="session")
+def write_dir():
+    write_dir = "tmp"
+    os.makedirs(write_dir, exist_ok=True)
+    yield Path(write_dir)
+    shutil.rmtree(write_dir)
+
+t = [1238163456,12345677890, 12345677890, 12345677890, 12345677890, 1238167552]
+y = [23, 53.5, 234, 0, 5]
+datasets = { 'dataset1': {'field_1': 'test1', 'field_2': 'test2', 'field_3': 'test3'},
+             'dataset2': {'field_4': 'test4', 'field_5': 'test5', 'field_6': 'test6'}}
+prefix = "out"
+
+def test_write_timeseries(write_dir:"Path"):
+    
+    tmpdir = write_dir 
```

Comment:
Is there a reason to rename the variable like this?

Reply:
Fixed this